Makefile.am

```diff
@@ -53,15 +59,21 @@ stream_DATA = $(NULL)
 recordingsdir = $(datadir)/janus/recordings
 recordings_DATA = $(NULL)
 
+if WINDOWS_OS
+INSTDIR = ../..
```

Comment:
Hi Lorenzo, I hope you had great holidays. Thank you for your comment. This actually entire feature that enables to use relative paths within Janus code. It enables to create a package that is path independent so that sources can be compiled it in e.g. c:\msys64\mingw64 and binaries can be then installed in selected path by the user e.g. C:\Program Files\Janus WebRTC Gateway